playbooks/roles/edxapp/templates/edx/app/edxapp/cms.sh.j2

```diff
@@ -29,6 +29,7 @@ export DD_DJANGO_INSTRUMENT_MIDDLEWARE=false
 # We want to be able to toggle this on separately from DD in general.
 {% if EDXAPP_DATADOG_PROFILING_ENABLE %}
 export DD_PROFILING_ENABLED=true
+export DD_PROFILING_STACK_V2_ENABLED=true
```

Comment:
So it looks like there's `DD_PROFILING_STACK_ENABLED` which defaults to true, and this one defaults to false. I guess this means that stack collection is a normal part of profiling, and this would change us to a v2 of that stack collection.